src/main/scala/rx/lang/scala/Notification.scala

```diff
@@ -73,7 +73,7 @@ sealed trait Notification[+T] {
  * import Notification._
  * Observable(1, 2, 3).materialize.subscribe(n => n match {
```

Comment:
Hi,
As per your request , I did the following  :
https://github.com/ReactiveX/RxScala/compare/0.x...elegoff:0.x

Thanks

Eric

--
Eric Le Goff
http://fr.linkedin.com/in/elegoff
@elegoff <https://twitter.com/elegoff>


On Wed, May 6, 2015 at 7:18 PM, Shixiong Zhu <notifications@github.com>
wrote:

> In src/main/scala/rx/lang/scala/Notification.scala
> <https://github.com/ReactiveX/RxScala/pull/157#discussion_r29780341>:
>
> > @@ -73,7 +73,7 @@ sealed trait Notification[+T] {
> >   * import Notification._
> >   * Observable(1, 2, 3).materialize.subscribe(n => n match {
>
> Could you also change Observable(1, 2, 3) to Observable.just(1, 2, 3)? Observable(1,
> 2, 3) does not work now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ReactiveX/RxScala/pull/157/files#r29780341>.
>